translations/monero-core_fr.ts

```diff
@@ -150,7 +150,7 @@
     <message>
         <location filename="../pages/History.qml" line="70"/>
         <source> selected: </source>
-        <translation> séléctionné : </translation>
+        <translation> sélectionné&nbsp;: </translation>
```

Comment:
Any special reason for changing all spaces to &nbsp;? Also wonder if the space between the word and the : needed anyway? Look at the source string.